src/helpers/id-generator.ts

```diff
@@ -0,0 +1,7 @@
+export function getDateShortString(): string {
```

Comment:
The function is great - a perfect example for helper function.

However the naming of the file threw me off. What is id-generator? Here we're not generating any id.